raiden/api/python.py

```diff
@@ -698,7 +698,12 @@ def set_total_channel_deposit(
 
         # set_total_deposit calls approve
         # token.approve(netcontract_address, addendum)
-        channel_proxy.set_total_deposit(total_deposit=total_deposit, block_identifier=blockhash)
+        try:
+            channel_proxy.set_total_deposit(
+                total_deposit=total_deposit, block_identifier=blockhash
+            )
+        except RaidenRecoverableError:
+            log.info("The deposit has already been done or the channel is no longer open.")
```

Comment:
Could you change this to something like `Deposit failed: {e}`? I think `already done` and `no longer open` are not the only reasons the deposit may fail.